CHANGELOG.md

```diff
@@ -1,4 +1,21 @@
 # Changelog
+## 2.0.1
```

Comment:
Patch versions are for backwards compatible bugfixes typically (https://semver.org).

In this case we're probably going for `2.1.0`. It's still a significant change but it doesn't break our public API.

Reply:
I think arguments could be made for this being a major or minor *or* patch version the more I think about it. Minor version still seems more sensible to me, preserving the patch version for more innocuous changes later.